changelog/april2024/2024-04-04-webhosting-added-all-regions-are-now-available.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: All regions are now available
+status: added
+author:
+    fullname: 'Join the #[xxx] channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-04-04
+category: managed-services
+product: webhosting
+---
+
+All 3 regions are now available for Web Hosting !
+ You can order your hosting in Paris, Amsterdam or Warsaw.
```

Comment:
Our Web Hosting services are now accessible in all three regions: Paris, Amsterdam, and Warsaw. 

Whether you are seeking reliability, speed, or localized service, you can now order your hosting from the location that best suits your needs.